run-health-checks.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+SKU=$( curl -H Metadata:true --max-time 10 -s "http://169.254.169.254/metadata/instance/compute/vmSize?api-version=2021-01-01&format=text")
+echo "Running health checks for $SKU SKU..."
```

Comment:
can we cast SKU to lower case, I don't know if this is always guaranteed to have the same capitalization. 

echo "$SKU" | tr '[:upper:]' '[:lower:]'